tools/tesseratos/include/tesseratos/toolbox/plugin.hpp

```diff
@@ -0,0 +1,55 @@
+
+
+/// @dir
+/// @brief @ref toolbox-plugin plugin directory.
+
+/// @file
+/// @brief Plugin entry point.
+/// @ingroup toolbox-plugin
+
+#pragma once
+
+#include <cubos/engine/cubos.hpp>
+
+namespace tesseratos
+{
+    /// @defgroup tesseratos-toolbox-plugin Toolbox
+    /// @ingroup tesseratos
+    /// @brief Adds a resource used to keep track of whether each tool is open or not.
+    ///
+    ///
+    /// ## Resources
+    /// - @ref Toolbox
+
+    /// @brief Resource which manages other tools windows.
+    class Toolbox final
+    {
+    public:
+        std::unordered_map<std::string, bool> toolsMap;
```

Comment:
This field should be private, you can move it to the bottom of the class definition, and prefix it with `m`. E.g:
```cpp
private:
     std::unordered_map<std::string, bool> mToolsMap;
};
```

Reply:
okay, im allowed to make other method(s) to be able iterate thru the map right? just making sure